code/game/turfs/space/space.dm

```diff
@@ -54,12 +54,12 @@
 				qdel(A)
 				return
 
-			var/specials_list = recursive_type_check(A, list(/obj/item/weapon/disk/nuclear, /mob/living/silicon/robot/mommi))
+			var/specials_list = search_contents_for(A, list(/obj/item/weapon/disk/nuclear, /mob/living/silicon/robot/mommi))

```

Comment:
Oh but it is

Arg 1: Search this item
Arg 2: If defined, objects of types not in this list are excluded (basically only search for these types)